src/python/CRABInterface/Regexps.py

```diff
@@ -103,7 +103,6 @@
 RX_DN = re.compile(r"^/(?:C|O|DC)=.*/CN=.")
 ## worker subresources
 RX_SUBPOSTWORKER = re.compile(r"^(state|start|failure|success|process|lumimask)$")
-RX_SUBGETWORKER = re.compile(r"jobgroup")
```

Comment:
@belforte this regex is used in https://github.com/dmwm/CRABServer/blob/8dcb7a5d4bc0ed4af0fd39ec9f7cc69a1d15f7cc/src/python/CRABInterface/RESTWorkerWorkflow.py#L55
since you removed this regex, i assume that this uses for it is absolete. can i remove this line?

Reply:
??? I think that you removed that regexp, not me: https://github.com/dmwm/CRABServer/pull/6831#pullrequestreview-794691213
too many change at same time ?

Reply:
My changes in #6831 are a consequence to this change of yours in this merged PR.
Since this PR removed `RX_SUBGETWORKER` from `CRABInterface/Regexps.py`, then in #6831 I had to removed it from the imports. It was failing otherwise.